src/js/media-poster-image.js

```diff
@@ -53,7 +74,10 @@ class MediaPosterImage extends window.HTMLElement {
     this.attachShadow({ mode: 'open' });
     this.shadowRoot.appendChild(template.content.cloneNode(true));
 
+    this.container = this.shadowRoot.querySelector('#container');
     this.image = this.shadowRoot.querySelector('#image');
+
+    this.image.addEventListener('load', () => loadFade(this.image));
```

Comment:
Christian was wondering whether this is necessary but I think it does, especially since I think we should expose the `loading` property in `media-poster-image` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading